_posts/2016-02-29-phishing.md

```diff
@@ -0,0 +1,54 @@
+
+### 1. Server-side Email filtering
+
+The first line of defense needed is anti-spam and email filtering for inbound email, to prevent emails from being received by your employees in the first place. Various technologies can be used including anti-spam filter based on BLs and IP reputation services, a server-side antivirus scan for attachments, and MX record/domain verification using Sender Policy Framework (SPF). Emails can be rejected by your mailserver so that they never make it to your users.
+
+### 2. User training and internal policies
+
+Inevitably, some emails are going to get through your email filters, no matter how infrequently. It is important to reduce the likelihood that employees are tricked into falling for the phishing scams by providing training as to how to recognise likely phishing attempts. It helps if advice is consistent - if users are aware that internal IT staff will never ask them for their passwords, then they are less likely to fall for a phishing attack asking them for their password. There are also excellent online tools such as [VirusTotal](https://www.virustotal.com/) which allow users to upload suspected attachments before opening them in order to check whether they contain any viruses.
+
+### 3. Mail client software
+
+Mail client software has a part to play in providing warning to users that opening attachments from emails or clicking on links may provide some element of risk of virus and phishing attacks and provide reminders to users of their training at the time that the phishing email is received.
+
+### 4. Spam email reporting
+
+Several mailservers and clients now support mechanisms whereby users can report emails as spam or viruses. In a corporate mailsystem, machine logic can be used to learn based on feedback from a few users when an email is spam, a phishing attempt or contains a virus, and actively move the mail item to trash or spam folders for users who have not yet read the mail, providing a fast, automatic and crowd-sourced approach to mail veracity.
+
+### 5. Internet Content Filtering (ICF) and restricted outbound internet access.
+
+A further measure that can help is internet content filtering on outbound web requests from employee workstations and laptops. ICF is a mechanism that intercepts outbound web requests and verifies whether they are to be permitted. Very often a phishing email will not have a malicious payload itself, in order to improve its chances of being delivered to targeted victims. It will instead contain a seemingly innocuous word document or Excel document attachment with a simple macro that executes on file open and attempts to download the real virus payload, or just a link to a website. Usually this file or webpage will be hosted on known malicious domains and ICF can actively block access to these domains so that the payload cannot be downloaded or form that attempts to capture their password cannot be loaded.
```

Comment:
Last sentence just needs a bit of finessing - maybe something like "Usually this file or webpage will be hosted on a domain known to be malicious. ICF can be used to actively block access to such domains, preventing the payload from being downloaded or the form designed to capture passwords from loading in the user's browser."